### PR TITLE
Icp ecosystem changes

### PR DESCRIPTION
**Pull request by Mohdniyam**

### CODE REVIEW

File: data/ecosystems/i/internet-computer.toml [hunk 1]
```diff
@@ -21728,3 +21728,248 @@ url = "https://github.com/NnsDao/nnsdao-fe"
...
+url = "https://github.com/nikhlu07/dbank"
+
+[[repo]]
+url = "https://github.com/Absar11/Dbank"
+
+[[repo]]
+url = "https://github.com/sameer152005/privad"
+
+[[repo]]
+url = "https://github.com/Preeti-2507/TO-DO"
```
Comment (eherrerosj) [inline data/ecosystems/i/internet-computer.toml:21882]: how's this repo using ICP?

File: data/ecosystems/i/internet-computer.toml [hunk 2]
```diff
@@ -21728,3 +21728,248 @@ url = "https://github.com/NnsDao/nnsdao-fe"
...
+
+[[repo]]
+url = "https://github.com/SURYANSHI-WEB/coven"
+
+[[repo]]
+url = "https://github.com/Sagarshah2004/Recipes"
```
Comment (eherrerosj) [inline data/ecosystems/i/internet-computer.toml:21888]: can probably remove

File: data/ecosystems/i/internet-computer.toml [hunk 3]
```diff
@@ -21728,3 +21728,248 @@ url = "https://github.com/NnsDao/nnsdao-fe"
...
+
+[[repo]]
+url = "https://github.com/AAshu1412/ICP-Blockchain-Calculator"
+
+[[repo]]
+url = "https://github.com/abhinav-nav/dakepper"
```
Comment (eherrerosj) [inline data/ecosystems/i/internet-computer.toml:21894]: how's this ICP?

**Review by eherrerosj — CHANGES_REQUESTED** [review]
left a few comments

**Comment by Mohdniyam** [comment]
Thank for reviewing @eherrerosj  , I have checked the repo list again and made the suggested changes as well as removed other unnecessary repos.

File: data/ecosystems/i/internet-computer.toml [hunk 4]
```diff
@@ -21729,1016 +21729,141 @@ url = "https://github.com/NnsDao/nnsdao-fe"
...
-url = "https://github.com/balbirs22/ICP_Blockchain"
-
-[[repo]]
-url = "https://github.com/Anshikarathour64/sat_blockchain"
-
-[[repo]]
-url = "https://github.com/diwakarsha/satworkshop"
-
-[[repo]]
-url = "https://github.com/AkshatBhatnagar29/blockbashhack"
```
Comment (eherrerosj) [inline data/ecosystems/i/internet-computer.toml:22702]: why is this repo being removed? It is actually ICP, no?

File: data/ecosystems/i/internet-computer.toml [hunk 5]
```diff
@@ -21729,1016 +21729,141 @@ url = "https://github.com/NnsDao/nnsdao-fe"
...
-
-[[repo]]
-url = "https://github.com/gurleenkaur164/gurleen"
-
-[[repo]]
-url = "https://github.com/Nishesh-singla/Blockchain"
```
Comment (eherrerosj) [inline data/ecosystems/i/internet-computer.toml:22708]: why is this repo being removed? It is using actually ICP, no?

File: data/ecosystems/i/internet-computer.toml [hunk 6]
```diff
@@ -21729,1016 +21729,141 @@ url = "https://github.com/NnsDao/nnsdao-fe"
...
-url = "https://github.com/Amritesh-Bhardwaj/ICP_Blockbash"
-
-[[repo]]
-url = "https://github.com/balbirs22/ICP_Blockchain"
-
-[[repo]]
-url = "https://github.com/Anshikarathour64/sat_blockchain"
-
-[[repo]]
-url = "https://github.com/diwakarsha/satworkshop"
```
Comment (eherrerosj) [inline data/ecosystems/i/internet-computer.toml:22699]: why is this repo being removed? It is using actually ICP, no?

File: data/ecosystems/i/internet-computer.toml [hunk 7]
```diff
@@ -21729,1016 +21729,141 @@ url = "https://github.com/NnsDao/nnsdao-fe"
...
-
-[[repo]]
-url = "https://github.com/AkshatBhatnagar29/blockbashhack"
-
-[[repo]]
-url = "https://github.com/gurleenkaur164/gurleen"
```
Comment (eherrerosj) [inline data/ecosystems/i/internet-computer.toml:22705]: why is this repo being removed? It is using actually ICP, no?

**Review by eherrerosj — CHANGES_REQUESTED** [review]
I checked a subset of the repos you are removing and I'm a bit confused on why they should be removed provided that they do build on top of ICP

**Comment by Mohdniyam** [comment]
@eherrerosj , I have reverted the unnecessary removal of repos, this happened due to not taking proper pull requests from master. Hope we are good to go now.

**Comment by jubos** [comment]
Manually merged with [bdfaebb](https://github.com/electric-capital/crypto-ecosystems/commit/bdfaebb9618738aada649e95a89ae2c7036fc7b0)

**Pull request closed without merging**